test/ctia/entity/event_test.clj

```diff
@@ -0,0 +1,242 @@
+                              :after "changed description"}}]}
+                          {:owner "user1",
+                           :groups ["group1"],
+                           :timestamp #inst "2042-01-01T00:00:00.000-00:00",
+                           :tlp "amber"
+                           :entity
+                           {:schema_version ctim-schema-version,
+                            :target_ref
+                            "http://localhost:3001/ctia/incident/incident-00000000-0000-0000-0000-111111111112",
+                            :type "relationship",
+                            :created "2042-01-01T00:00:00.000Z",
+                            :modified "2042-01-01T00:00:00.000Z",
+                            :source_ref
+                            "http://localhost:3001/ctia/casebook/casebook-00000000-0000-0000-0000-111111111117",
+                            :id
+                            "http://localhost:3001/ctia/relationship/relationship-00000000-0000-0000-0000-111111111119",
+                            :tlp "amber",
+                            :groups ["group1"],
+                            :owner "user1",
+                            :relationship_type "related-to"},
+                           :id
+                           "http://localhost:3001/ctia/event/event-00000000-0000-0000-0000-111111111120",
+                           :type "event",
+                           :event_type :record-created}
+                          {:owner "user1",
+                           :groups ["group1"],
+                           :timestamp #inst "2042-01-01T00:00:00.000-00:00",
+                           :tlp "amber"
+                           :entity
+                           {:description "changed description",
+                            :schema_version ctim-schema-version,
+                            :type "incident",
+                            :created "2042-01-01T00:00:00.000Z",
+                            :modified "2042-01-02T00:00:00.000Z",
+                            :incident_time {:opened "2042-01-01T00:00:00.000Z"},
+                            :status "Open",
+                            :id
+                            "http://localhost:3001/ctia/incident/incident-00000000-0000-0000-0000-111111111112",
+                            :tlp "amber",
+                            :groups ["group1"],
+                            :confidence "High",
+                            :owner "user1"},
+                           :id
+                           "http://localhost:3001/ctia/event/event-00000000-0000-0000-0000-111111111121",
+                           :type "event",
+                           :event_type :record-deleted}]
+                         (:parsed-body (get (str "ctia/event/search?query=" q)
+                                            :content-type :json
+                                            :headers {"Authorization" "user1"})))))))))))))))
```

Comment:
_remark_: I think if you could you should use Matt strategy to improve the readability and efficiency of such tests by providing a list of smaller checks instead of an equality with a very big object. Typically, check the count of element is correct. Check each element `:type` is equal to `"event"`, etc...